packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -41,7 +41,7 @@
     <template #select="{ item }">
       <oc-checkbox
         :id="`resource-table-select-${resourceDomSelector(item)}`"
-        :label="getResourceCheckboxLabel(item)"
+        :label="$gettext('Select %{ type }', { type: item.type })"
```

Comment:
Breaks translations ;-) `"Wähle folder aus"` would be the german result, as the type is not translated.

Reply:
Arghs. Then this is going to be ugly 😄